<?xml version="1.0" ?>
<launch>
  <!--
      Launch file for the admittance controller
  -->

  <rosparam file="$(find manos_velocity_command)/config/manos_velocity_command.yaml" command="load"/>

  <node pkg="manos_velocity_command" name="velocity_controller"
  type="velocity_controller_node" output="screen"/>

</launch>
